alerting/src/test/kotlin/org/opensearch/alerting/resthandler/MonitorRestApiIT.kt

```diff
@@ -887,23 +889,62 @@ class MonitorRestApiIT : AlertingRestTestCase() {
         assertEquals("Scheduled job is not enabled", false, responseMap[ScheduledJobSettings.SWEEPER_ENABLED.key])
         assertEquals("Scheduled job index exists but there are no scheduled jobs.", false, responseMap["scheduled_job_index_exists"])
         val _nodes = responseMap["_nodes"] as Map<String, Int>
-        assertEquals("Incorrect number of nodes", numberOfNodes, _nodes["total"])
-        assertEquals("Failed nodes found during monitor stats call", 0, _nodes["failed"])
-        assertEquals("More than $numberOfNodes successful node", numberOfNodes, _nodes["successful"])
+        validateAlertingStatsNodeResponse(_nodes)
+    }
+
+    fun `test monitor stats when disabling and re-enabling scheduled jobs with existing monitor`() {
+        // Enable Monitor jobs
+        enableScheduledJob()
+        val monitorId = createMonitor(randomQueryLevelMonitor(enabled = true), refresh = true).id
+
+        var alertingStats = getAlertingStats()
+        assertEquals("Scheduled job is not enabled", true, alertingStats[ScheduledJobSettings.SWEEPER_ENABLED.key])
+        assertEquals("Scheduled job index does not exist", true, alertingStats["scheduled_job_index_exists"])
```

Comment:
Use `assertTrue` instead

Reply:
Leaving this as `assertEquals` because `assertTrue` doesn't infer type and `alertStats` is a Map<String, Any> so all the actual items being passed in would need to have `as Boolean` added to them (it complains otherwise) which would get a little bothersome. I think that's why the other tests wrote it that way as well.